src/transformers/models/funnel/modeling_tf_funnel.py

```diff
@@ -185,7 +185,7 @@ def init_attention_inputs(self, inputs_embeds, attention_mask=None, token_type_i
         # inputs_embeds has shape batch_size x seq_len x d_model
         # attention_mask and token_type_ids have shape batch_size x seq_len
         self.pooling_mult = 1
-        self.seq_len = seq_len = inputs_embeds.shape[1]
+        self.seq_len = seq_len = shape_list(inputs_embeds)[1]
```

Comment:
I always thought we need to use `shape_list` instead of `.shape()` to prevent bugs of dynamic vs. static shapes -> is this not the case anymore?